flask/app/families.py

```diff
@@ -245,6 +304,7 @@ def update_family(id: int):
             }
         )
     except:
+        app.logger.debug("Update unsuccessful")  # under what cases would this fail
```

Comment:
Cases like these would be a prime example for catching the exception and logging it.